activitychart-basic.html

```diff
@@ -0,0 +1,274 @@
+<link rel="import" href="d3-import.html">
+
+
+<!--
+`activitychart-basic`
+This component shows an activity history, similar to the colored github chart
+
+@demo demo/activitychart_basic_demo.html
+-->
+<dom-module id="activitychart-basic">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <iron-ajax
+        id="ajaxCaller"
+        url="[[dataurl]]"
+        handle-as="json"
+        on-response="handleResponse"
+        debounce-duration="300">
+    </iron-ajax>
+    <div style="display:none;">{{dataset}}</div>
+    <div id="diagram"></div>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'activitychart-basic',
+
+      properties: {
+        /** URL of data to be plotted */
+        dataurl: {
+          type: String,
+          value: '',
+        },
+        /** y values in JSON format (data to be displayed) */
+        dataset: {
+          type: Array,
+        },
+        /** x values in JSON format (labels for data) */
+        labels: {
+          type: Array,
+        },
+        /** Primary color of chart; formats supported are Hex, Short Hex, RGB (absolute), RGB (percentual), RGBA (absolute), RGBA (percentual), keyword (e.g. 'white'), HSL and HSLA */
+        primarycolor: {
+          type: String,
+          value: '#930517',
+        },
+        /** Accent color of chart; formats supported are Hex, Short Hex, RGB (absolute), RGB (percentual), RGBA (absolute), RGBA (percentual), keyword (e.g. 'white'), HSL and HSLA  */
+        accentcolor: {
+          type: String,
+          value: '#F5E8BB',
+        },
+        /** Number of colored dots per column (opt, default 8) */
+        squaresPerColumn: {
+          type: Number,
+          value: 8,
+        },
+        /** Width of square (opt, default: 15) */
+        squareWidth: {
+          type: Number,
+          value: 15,
+        },
+        /** Spacing of squares (opt, default: 20) */
+        squareSpacing: {
+          type: Number,
+          value: 20,
+        },
+        //this could also be solved by setting value to a function calculating the default min/max value from the dataset (but only if we receive the dataset as a JSON object isntead of just receiving a URL)
+        /** Minimum value from the dataset that is used for the color scale can be overwritten. */
+        minValue: {
+          type: Number,
+          value: null
+        },
+        /** Maximum value from the dataset that is used for the color scale can be overwritten. */
+        maxValue: {
+          type: Number,
+          value: null
+        },
+        /** Labels for x and y axis.
+        Please provide as {'x': {'label': String}, 'y': {'label': String, 'values': String[]}}
+        The correct labels for the x axis are chosen programmatically based on the data.*/
+        axisLabels: {
+          type: Object,
+          value: {
+            "x": {
+              "label": "Tag"
```

Comment:
probably better if we don't hardcode "Tag" and "Stunde", as this may change if the language is switched